shelley/pool-ranking/pool-ranking.tex

```diff
@@ -265,10 +304,19 @@ \section{Ranking and Saturation}
     \end{array}
     \right.
 \]
-
-Finally, users can use $r_\mathrm{member, nm}$ to select a pool
+Delegators can use $r_\mathrm{member, nm}$ to select a pool
 that maximizes their rewards.
 
+\section{Haskell Source Code}
+
+\begin{comment}
+  Include the Haskell source code here.
```

Comment:
isn't there a problem of code drift? maybe we could just say "this is the code at the time of writing"?